tests/acceptance/PreviewPostNotificationNewsletterCest.php

```diff
@@ -18,4 +18,33 @@ public function previewStandardNewsletter(\AcceptanceTester $i) {
     $i->switchToNextTab();
     $i->waitForElement('.mailpoet_template');
   }
+
+  public function previewPostNotificationNewsletterInEditor(\AcceptanceTester $i) {
+    $i->wantTo('Preview and send post notification newsletter inside editor');
+    $newsletterName = 'Preview in Browser Post Notification';
+    $newsletter = new Newsletter();
+    $newsletter->withSubject($newsletterName)->withPostNotificationsType()->create();
+    $i->login();
+    $i->amOnMailpoetPage('Emails');
+    $i->waitForText('Emails');
+    $i->click('Post Notifications', '[data-automation-id="newsletters_listing_tabs"]');
+    $i->waitForText($newsletterName);
+    $i->clickItemRowActionByItemName($newsletterName, 'Edit');
+    $i->click('[data-automation-id="sidebar_preview_region_heading"]');
+    $i->waitForText('Send preview');
+    $i->click('Send preview');
+    $i->waitForText('Your test email has been sent!');
+    $i->click('.notice-dismiss');
+    $i->clearField('#mailpoet_preview_to_email');
+    $i->click('Send preview');
+    $i->waitForText('Enter an email address to send the preview newsletter to.');
+    $i->fillField('#mailpoet_preview_to_email', 'test2@test.com');
+    $i->click('Send preview');
+    $i->waitForText('Your test email has been sent!');
```

Comment:
Why do we need to send the preview email twice? Looks like the first one may be eliminated if I'm not missing something. 

It also might make sense to check that a preview email was received using the `checkEmailWasReceived` method, see examples in other test cases.

Reply:
Regarding twice sending check - we had issue in the past that only sending to one email was possible, now with this check we should cover it.
Thanks for your suggestion, I have added `checkEmailWasReceived` at the end of both tests, didn't want to include it in the middle of testing, I think at the end is just fine. Also, I added comments in my tests, so now it is more clear for everyone.